blues/simulation.py

```diff
@@ -56,16 +69,32 @@ def generateAlchSystem(self, system, atom_indices):
             The OpenMM System object corresponding to the reference system.
         atom_indices : list
             Atom indicies of the move.
+        freeze_center : str
+            AmberMask selection for the center in which to select atoms for zeroing their masses. Default: LIG
+        freeze_distance : float
+            Distance (angstroms) to select atoms for retaining their masses. Atoms outside the set distance will have their masses set to 0.0. Default: 5.0
+        freeze_solvent : str
+            AmberMask selection in which to select solvent atoms for zeroing their masses. Default: HOH,NA,CL
         """
-        import logging
         logging.getLogger("openmmtools.alchemy").setLevel(logging.ERROR)
         factory = alchemy.AbsoluteAlchemicalFactory(disable_alchemical_dispersion_correction=True)
         alch_region = alchemy.AlchemicalRegion(alchemical_atoms=atom_indices)
         alch_system = factory.create_alchemical_system(system, alch_region)
+
+
+        if freeze_distance:
+            #Atom selection for zeroing protein atom masses
+            mask = parmed.amber.AmberMask(self.structure,"(:%s<:%f)&!(:%s)" % (freeze_center,freeze_distance,freeze_solvent))
+            site_idx = [i for i in mask.Selected()]
+            logger.info('Zeroing mass of %s atoms %.1f Angstroms from LIG' % (len(site_idx), freeze_distance))
+            alch_system = self._zero_allother_masses(alch_system, site_idx)
+        else:
+            pass
+
         return alch_system
 
     def generateSystem(self, structure, nonbondedMethod='PME', nonbondedCutoff=10,
-                       constraints='HBonds', **opt):
+                       constraints='HBonds', hydrogenMass=None, **opt):
```

Comment:
Why `hydrogenMass=None`?

Reply:
(This probably means a doc string update...)

Reply:
`hydrogenMass=None` is the default when creating a parmed.Structure. So this means this does NOT do HMR. I can add docstrings yes.

https://parmed.github.io/ParmEd/html/structobj/parmed.structure.Structure.html#parmed.structure.Structure.createSystem